src/plugins/intel_cpu/src/nodes/reorder.cpp

```diff
@@ -262,17 +254,16 @@ void Reorder::createReorderPrimitive(const dnnl::memory::desc& srcDesc,
     // perform such conversion if the source tensor can be reshaped to the destination rank. This is
     // useful in situations when rank in IR does not much rank that is required by the oneDNN primitive,
     // but the input tensor can be reshaped (e.g. weights for grouped convolutions, biases etc.)
-    if (src_blocked->getDesc().hasLayoutType(LayoutType::ncsp) &&
-        src_blocked->GetShape().getRank() != dst_blocked->GetShape().getRank()) {
-        const auto newDims = dst_blocked->getStaticDims();
+    auto &srcMemPtr = getParentEdgeAt(0)->getMemoryPtr();
```

Comment:
Changing the method signature allow get rid of requesting memory objects from edges.

Reply:
fixed, thank you Maksim.